variables.tf

```
@@ -3,6 +3,12 @@ variable "account" {
   description = "The name of the account"
 }
 
+variable "create_email_address" {
+  type        = bool
+  default     = true
+  description = "Boolean that can be used for disabling the creation of the email address"
```

We prefer to use `Set to true to...` syntax:

```suggestion
  description = "Set to true to create the o365 email alias"
```